tests/core/test_schema_loader.py

```diff
@@ -107,20 +107,20 @@ def test_create_external_models(tmpdir, assert_exp_eq):
     assert context.models['"memory"."sushi"."raw_fruits"'].gateway is None
 
 
-def test_gateway_specific_external_models(tmpdir: Path):
+def test_gateway_specific_external_models(tmp_path: Path):
```

Comment:
The `tmpdir` fixture returns some kind of Pytest internal Path object which didnt play nice with `yaml.load()` which wants a `pathlib.Path` object.

So I switched to the `tmp_path` fixture